app/views/nic/_base_form.html.erb

```diff
@@ -31,6 +31,8 @@
                  :size => "col-md-8", :label_size => "col-md-3" } %>
   <%= nic_subnet_field f, :subnet_id, :"Subnet::Ipv4",
                 { :label => _("IPv4 Subnet"),
+                :label_help => _("IPv4 Subnet with TFTP Capsule is required for PXE based provisioning.").html_safe,
```

Comment:
This doesn't need to be marked as `html_safe` as it doesn't contain any HTML. Additionally, the correct term is smart proxy, not capsule.
```suggestion
                :label_help => _("IPv4 subnet with associated TFTP smart proxy is required for PXE based provisioning."),
```